src/surfaces/drawer/drawer.d.ts

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import DrawerActionBar from './drawerActionBar';
+import DrawerContainer from './drawerContainer';
+import DrawerContent from './drawerContent';
+import DrawerDataCards from './drawerDataCards';
+import DrawerDataGrid from './drawerDataGrid';
+import DrawerDataGroups from './drawerDataGroups';
+import DrawerDeprecatedFiltersDrawer from '../drawerDeprecated/drawerDeprecatedFiltersDrawer';
+import DrawerDetailsWindow from './drawerDetailsWindow';
+import DrawerFiltersDrawer from './drawerFiltersDrawer';
+import DrawerFiltersRail from './drawerFiltersRail';
+import DrawerNavigation from './drawerNavigation';
+import DrawerTitleBar from './drawerTitleBar';
+import DrawerWing from './drawerWing';
+
+export interface DrawerPropTypes {
+    /**
+     * The content of the Drawer.
+     */
+    children?: React.ReactNode;
+    /**
+     * Override or extend the styles applied to the Drawer.
+     */
+    classes?: {
+        root: string;
+    };
+    /**
+     * Add additional classes to the Drawer.
+     */
+    className?: string;
+    /**
+     * Used for DOM testing. https://testing-library.com/docs/queries/bytestid/
+     */
+    dataTestId?: string;
+    dimmer?: boolean;
+    isOpen: boolean;
+    isModal?: boolean;
+    maxHeight?: number;
+    maxWidth?: number | string;
+    onClickOutside?: Function;
+    onClose?: Function;
+    onCloseComplete?: Function;
+    onOpenComplete?: Function;
+    positionX?: 'left' | 'right';
+    positionY?: 'bottom' | 'top';
+    positionYOffset?: number;
+    shadowSize?: 'large' | 'small' | 'xsmall';
+    /**
+     * Supply any inline styles to the Drawer.
+     */
+    style?: {};
+    theme?: {
+        zIndex: any;
+    };
+    wing?: {};
```

Comment:
Do we not want `React.ReactNode` here?

Reply:
Fixed.

Reply:
https://github.com/saddlebackdev/react-cm-ui/pull/346/commits/373dffa65bef7979afbf129396dec1f44990425d#diff-e5aff959e716fbaa989caeca50b87bbffa93305e437f68512ca9944fa92ebdbdL55-R57
Resolving.